server/bin/pbench-audit-server.py

```diff
@@ -194,7 +197,10 @@ def verify_archive(hier):
             elif os.path.islink(item):
                 symlink_item = f"{base_item} -> {os.path.realpath(item)}"
                 hier.add_controller_link(controller, symlink_item)
-            elif os.path.isfile(item) and not base_item.endswith('.tar.xz') and not base_item.endswith('.tar.xz.md5'):
+            elif all([os.path.isfile(item), not base_item.endswith('.tar.xz'), not base_item.endswith('.tar.xz.md5')]):
+                if hidden_entries:
```

Comment:
This is misplaced: you need to deal with hidden objects in a separate loop (basically, any hidden file other than ".prefix" is an unexpected object).